web/public/js/controllers/manage-dojo-users-controller.js

```diff
@@ -60,11 +62,243 @@ function cdManageDojoUsersCtrl($scope, $state, $q, cdDojoService, alertService,
     $scope.loadPage(false);
   };
 
+  $scope.selectedItems = [];
+
+  $scope.showActionBar = false;
+
+  $scope.$watch('selectedItems', function (newValue) {
```

Comment:
Not sure if we want this. Clears the badge selection and disables the award badge button when selecting a different user